test/http-api/interface.js

```diff
@@ -21,8 +21,23 @@ describe('interface-ipfs-core over ipfs-http-client tests', () => {
     skip: { reason: 'TODO: DAG HTTP endpoints not implemented in js-ipfs yet!' }
   })
 
-  tests.dht(defaultCommonFactory, {
-    skip: { reason: 'TODO: DHT is not implemented in js-ipfs yet!' }
+  tests.dht(CommonFactory.create({
+    spawnOptions: {
+      initOptions: { bits: 512 }
+    }
+  }), {
+    skip: [
+      // dht.get
+      {
+        name: 'should get a value after it was put on another node',
+        reason: 'Needs https://github.com/ipfs/interface-ipfs-core/pull/383'
+      },
+      // dht.findprovs
+      {
+        name: 'should take options to override timeout config',
+        reason: 'Returns empty result'
```

Comment:
Would you mind clarifying why this is skipped?

Reply:
fixed in https://github.com/ipfs/interface-ipfs-core/pull/414 and will remove the skip!